alexthompson207/overlook
# Feature/basic html structure

# Pull Request

### What's this PR do?

Create the basic styling and HTML for login, guest landing dashboard, and guest search view.

### Where should the reviewer start?

In the `index.html` file and all the scss files.

### How should this be manually tested?

open the website locally, play around with adding and remove hidden class from different sections to see the different views.

### Any background context you want to provide?

N/a

### Screenshots (if appropriate):

### Questions:

Do I need a hidden class and it's display property for every scss file?
